dashboard/app/controllers/test_controller.rb

```diff
@@ -9,6 +9,13 @@ def hidden_script_access
     head :ok
   end
 
+  def levelbuilder_access
+    return unless (user = current_user)
+    user.permission = UserPermission::LEVELBUILDER
+    user.save!
+    head :ok
+  end
```

Comment:
Does this route, which allows any user in an environment where TestController is loaded, have implications for scripts under NDA?